### DISCUSSION TITLE
Page layout

### DISCUSSION THREAD

**Discussion by dwsmith1983** [dwsmith1983/recommend-resources#20] [Ideas]
How should we best layout the pages?

1. We could do pages by media format: book, article, video, etc
2. Split by topics
3. Some other good idea

@beljun @lukkiddd what are your thoughts/opinions on this?

Additionally, how would we want to nest these? Recommendations sub-directory in posts? Then we could create other content that doesn't co-mingle with recommendations.

**Comment by dwsmith1983** [comment]
I created a dev/layout-refactor if you two want to check it locally first. I would like to add some articles and more videos first before PR-ing if everyone likes this layout.

Feel free to add some content or argue against this style.

@beljun @lukkiddd

**Reply by lukkiddd** [reply]
Personally, I think split by topics would be helpful when someone looking for a specific area of interest.

**Reply by lukkiddd** [reply]
But for now, one page may enough since there is not much content yet.

**Reply by dwsmith1983** [reply]
Topics might be too small though as compared to media formats and we create many more subgroups.

**Reply by dwsmith1983** [reply]
Additionally, @beljun has a lot of content on hand. If he adds that, it would greatly increase the number of entries.

**Reply by lukkiddd** [reply]
Can't wait to absorb those content haha 🕺🏼 

Media format is ok as well, it would be something like videos, blogs, books.

**Reply by dwsmith1983** [reply]
Yeah. Pull the dev branch and build locally. You can see how it is structured.

**Reply by lukkiddd** [reply]
See it already, reviewed the PR as well
